project2/styles.css

```
@@ -0,0 +1,188 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    color: #686868;
+    font-size: 18px;
+}
+
+section {
+    background-color: #faf4e3;
+    max-width: 640px;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+header {
+    background-image: url(images/shrimps_2.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 300px;
+    border-radius: 5px;
+
+}
+
+.title {
+    display: block;
+    padding-left: 30px;
+    padding-top: 20px;
+}
+
+
+div {
+    height: 258px;
+    padding-left: 10px;
+}
+
+h2 {
+    text-align: left;
+    font-size: 42px;
+    color: #223b12;
+    font-family: "Notos Sans JP", sans-serif;
+    font-weight: 700;
+    margin-bottom: 10px;
+}
+
+.subhead {
+    text-align: left;
+    padding-left: 30px;
+}
+
+h3 {
+    text-align: right;
+    font-size: 22px;
+    font-family: "Notos Sans JP", sans-serif;
+    font-weight: 700;
+    padding-right: 30px;
+}
+
+h3:hover {
+    color: #223b12;
+    transition: 400ms;
+    text-decoration: underline;
+}
+
+h4 {
+    text-align: left;
+    font-size: 22px;
+    font-family: "Notos Sans JP", sans-serif;
+    font-weight: 700;
+    margin-top: 0px;
```

Zero measurements don't need units:
```suggestion
    margin-top: 0;
```